istioctl/cmd/kubeinject.go

```diff
@@ -367,7 +529,8 @@ kube-inject on deployments to get the most up-to-date changes.
 		fmt.Sprintf("ConfigMap name for Istio mesh configuration, key should be %q", configMapKey))
 	injectCmd.PersistentFlags().StringVar(&injectConfigMapName, "injectConfigMapName", defaultInjectConfigMapName,
 		fmt.Sprintf("ConfigMap name for Istio sidecar injection, key should be %q.", injectConfigMapKey))
-
+	injectCmd.PersistentFlags().StringVar(&mutatingWebhookConfigurationName, "mutatingWebhookConfigurationName", defaultInjectWebhookConfigName,
```

Comment:
Too long.  Perhaps just `--webhook`?

Reply:
Used `--webhookConfig` since it is a configuration